tests/ci/profiles/tf_cluster_profile.yml

```diff
@@ -44,7 +45,7 @@
     proxy: true
     labeling: true
     tagging: true
-    channel_group: staging
+    channel_group: stable
```

Comment:
why stable. We want to check the releases ASAP

Reply:
there is no channel_group named "staging", only stable

Reply:
Sorry, I've meant "candidate"

Reply:
changed to candidate